### ISSUE TITLE
Coffeescript support for Snippets

### ISSUE THREAD

**Issue by cdloh** [NicoSantangelo/sublime-jasmine#12]
Any chance that you could add support for coffeescript syntax for the snippets?

**Comment by NicoSantangelo** [comment]
I think it shouldn't be a problem, I'll have to give the `Change quote` feature a try and if it still works I'll add it to master.

I'll tackle this next week, but feel free to try it out and add a PR if you want 😄

**Comment by cdloh** [comment]
Any documentation on that feature? I can't see it in the ST3 Documentation.

**Comment by NicoSantangelo** [comment]
Not a lot, but in you can find it in the README [here](https://github.com/NicoSantangelo/sublime-jasmine#toggle-quotes) and in the code [here](https://github.com/NicoSantangelo/sublime-jasmine/blob/master/jasmine_commands.py#L101)

**Comment by NicoSantangelo** [comment]
@cdloh Hey! I know it's been a while but I finally got some time to tackle this. I'm using the branch [coffeescript-support](https://github.com/NicoSantangelo/sublime-jasmine/tree/coffeescript-support). I'm almost done but I need to test the package behaviour on update first.

I'll make a release when I finish 😄

**Comment by NicoSantangelo** [comment]
@cdloh Hello there!

I actually finished implementing this, but I realised that it's not a great idea because [Jasmine CoffeeScript](https://packagecontrol.io/packages/Jasmine%20CoffeeScript) already exists. So I think it's best if you install that package instead, to avoid adding collisions (and potential confusion) to the users of this package.

I'm sorry I left you waiting like this, and thanks a lot for the suggestion.
Let me know if I can help you with anything else!
